app/server/lib/SQLiteDB.ts

```diff
@@ -467,10 +478,32 @@ export class SQLiteDB implements ISQLiteDB {
     return metadata;
   }
 
+  /**
+   * Call this if you'd like the next write to be held until unpause() is
+   * called. Used by the backup system.
+   */
+  public pause() {
+    if (this._paused || this._closed) { return; }
```

Comment:
I don't see `this._paused` getting unset anywhere, so I think this logic isn't working the way it's intended to?

Reply:
Thank you for catching this! Added an unset and a test for pause/unpause.